app/api/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
 
     testApi("org.jetbrains.kotlin:kotlin-test") // Kotlin Test
     testApi("org.springframework.boot:spring-boot-starter-test") // Spring Test
+    testImplementation("org.mockito.kotlin:mockito-kotlin:4.1.0")
 }
 
 tasks.register<Copy>("copy-dev") {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, but no patch was found that overlapped with it. Original lines [182-182]

LG


<!-- This is an auto-generated comment by OSS CodeRabbit -->